Extensions/Settings/Dnn.PersonaBar.SqlConsole/admin/personaBar/scripts/SqlConsole.js

```diff
@@ -355,6 +358,10 @@ define(['jquery',
                             rightData = rightData.toLowerCase();
                         }
 
+                        if(isNaN(leftData) && isNaN(rightData)) {
+                            return 0;
+                        }
+
                         if (leftData === rightData) {
                             return 0;
                         } else if (leftData < rightData) {
```

Comment:
We also have a problem here. I noticed that somewhere between getting data from the server and sorting boolean `true` value gets changed into a string `"true"`. It also breaks the comparison which happens to be like `false < "true"` and `"true" < false" instead of proper bool comparison.